src/management/marketing.c

```diff
@@ -96,7 +96,7 @@ void marketing_update()
 		if (campaign == ADVERTISING_CAMPAIGN_RIDE_FREE || campaign == ADVERTISING_CAMPAIGN_RIDE) {
 			rct_ride* ride = GET_RIDE(campaignItem);
 			RCT2_GLOBAL(RCT2_ADDRESS_COMMON_FORMAT_ARGS, uint16) = ride->name;
-			RCT2_GLOBAL(0x013CE954, uint32) = ride->name_arguments;
+			RCT2_GLOBAL(RCT2_ADDRESS_COMMON_FORMAT_ARGS + 2, uint32) = ride->name_arguments;
```

Comment:
As seen throughout this PR, there is another `uint32` at `RCT2_ADDRESS_COMMON_FORMAT_ARGS + 4`, this probably should be `uint16` or something else is wrong here.